hack/generator/pkg/astbuilder/assignments.go

```diff
@@ -42,3 +42,50 @@ func SimpleAssignmentWithErr(lhs dst.Expr, tok token.Token, rhs dst.Expr) *dst.A
 		},
 	}
 }
+
+// SimpleIf creates a simple if statement with a single statement in each branch
+//
+// if <condition> {
+//     <trueBranch>
+// } else {
+//     <falseBranch>
+// }
+//
+func SimpleIf(condition dst.Expr, trueBranch dst.Stmt, falseBranch dst.Stmt) *dst.IfStmt {
```

Comment:
Maybe `SimpleIfElse`? (In contrast to a `SimpleIf` which wouldn't have an `else` clause)?